mix.exs

```diff
@@ -26,8 +26,9 @@ defmodule Binance.MixProject do
   defp deps do
     [
       {:httpoison, "~> 1.0"},
-      {:poison, "~> 4.0.0"},
```

Comment:
Wondering why we need to downgrade here

Reply:
This is not final code. There is a dependency conflict, I temporarily downgrade it to make it work so I could focus on implementing the main thing. Will fix it soon.